app/views/shelters/show.html.erb

```diff
@@ -7,3 +7,13 @@
   <p><%= @shelter.zip %></p>
   <%= link_to "Update #{@shelter.name}", "/shelters/#{@shelter.id}/edit" %> /
   <%= link_to "Delete #{@shelter.name}", "/shelters/#{@shelter.id}", method: :delete %>
+
+
+  <% @reviews.each do |review| %>
+    <%= review.title %>
+    <%= review.rating %>
+    <%= review.content %>
+    <% if review.image != "https://i.pinimg.com/originals/d1/21/f6/d121f64f59f7599486e3762fcd2f1475.jpg" %>
```

Comment:
the review `image` is only displayed if it was specified by the user upon creation